### PR TITLE
[infinite-list] Fixed ScrollHeight null issue

### PR DESCRIPTION
**Pull request by lokesh-0813**
### Summary
Fixes `Cannot read property 'scrollHeight' of null` issue when infinite list is unmounted immediately after browser resize.

Solves : #835

### CODE REVIEW

File: packages/terra-infinite-list/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
```
Comment (avinashg1994) [inline packages/terra-infinite-list/CHANGELOG.md]: nit: i think this should be in the "FIxed" section.

File: packages/terra-infinite-list/src/InfiniteList.jsx [hunk 2]
```diff
@@ -234,6 +234,7 @@ class InfiniteList extends React.Component {
     if (!this.contentNode) {
       return;
     }
+    clearTimeout(this.timer);
```
Comment (ShettyAkarsh) [inline packages/terra-infinite-list/src/InfiniteList.jsx]: @lokesh-0813, could let me know how this callout to cleartimeout will make the scrollheight value not null ? or is this call prevent certain calls which was happening before ?

Comment (lokesh-0813) [reply packages/terra-infinite-list/src/InfiniteList.jsx]: @ShettyAkarsh, scrollHeight is used in handleResize() which is called inside setTimeout() and when clearTimeout() is called, it terminates setTimeout().

File: packages/terra-infinite-list/CHANGELOG.md (hunk 1 above)
Comment (lokesh-0813) [reply packages/terra-infinite-list/CHANGELOG.md]: Updated now. Thank you.

**Pull request merged**